ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,72 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project.
+ It deploys Kubernetes with the Antrea image and tests whether the
+ OVS has hardware offload capabilities or not.
+
+## Workflow
+
+The CI tests run on a single node cluster with a ConnectX5 NIC card,
+ the latest stable Kubernetes binaries are built each time the scripts
+ run. In addition to the Kubernetes and the Antrea, the scripts deploy
```

Comment:
s/In addition to the Kubernetes and the Antrea/In addition to Kubernetes and Antrea

Reply:
Done